### PR TITLE
Update realdebrid.py

### PR DESCRIPTION
**Pull request by Pukabyte**
- Remove Sample Files From Being Selected
- Only Select Wanted Formats

### CODE REVIEW

File: backend/program/realdebrid.py [hunk 1]
```diff
@@ -118,13 +116,7 @@ def chunks(lst, n):
...
                     for container in containers:
-                        wanted_files = {}
-                        if item.type == "movie" and all(file["filesize"] > 200000 for file in container.values()):
-                            wanted_files = container
-                        if item.type == "season" and all(any(episode.number in parser.episodes_in_season(item.number, file["filename"]) for file in container.values()) for episode in item.episodes):
-                            wanted_files = container
-                        if item.type == "episode" and any(item.number in parser.episodes_in_season(item.parent.number, episode["filename"]) for episode in container.values()):
-                            wanted_files = container
+                        wanted_files = self._filter_wanted_files(container)
                         if len(wanted_files) > 0 and all(item for item in wanted_files.values() if Path(item["filename"]).suffix in WANTED_FORMATS):
```
Comment (Gaisberg) [inline backend/program/realdebrid.py:120]: Remove the additional check

**Review by Gaisberg — CHANGES_REQUESTED** [review]
minor comment, looks good.

File: backend/program/realdebrid.py [hunk 2]
```diff
@@ -146,27 +138,39 @@ def _set_file_paths(self, item):
...
+        wanted_files = self._filter_wanted_files(episode.active_stream.get("files"))
+        if wanted_files:
+            file_info = next(file_info for file_info in wanted_files.values() if episode.number in parser.episodes_in_season(episode.parent.number, file_info["filename"]))
+            episode.set("folder", episode.active_stream.get("name"))
+            episode.set("alternative_folder", episode.active_stream.get("alternative_name"))
+            episode.set("file", file_info["filename"])
+
+    def _filter_wanted_files(self, files):
+        """Filter unwanted files based on file extension and exclude 'sample' variations"""
+        unwanted_keywords = ["sample"]
```
Comment (dreulavelle) [inline backend/program/realdebrid.py:165]: should do by size instead.. theres too many keywords we'd need to match.. better off cutting off anything under X filesize, which can be set by user. 100mb default.

File: backend/program/realdebrid.py (hunk 1 above)
Comment (Pukabyte) [reply backend/program/realdebrid.py:120]: Which do I remove? lol
lines 125 -127?

**Comment by dreulavelle** [comment]
make sure to pull latest.. I pushed an update to dev

**Pull request closed without merging**